src/main/java/cz/muni/fi/pa165/library/services/SingleLoanService.java

```diff
@@ -8,25 +8,51 @@
 
 /** @author Martin Páleník 359817
  *  based on http://zetcode.com/springboot/crudrepository/
+ *
+ *  This represents the business logic of the
+ *  loans of book or books by a member.
  */
 
 @Service
 public class SingleLoanService {
+
     private SingleLoanRepository singleLoanRepository;
 
+    /**
+     * Constructor creates single loan between a member and a book
+     * @param singleLoanRepository
+     */
     public SingleLoanService(SingleLoanRepository singleLoanRepository) {
         this.singleLoanRepository = singleLoanRepository;
     }
 
+    /**
+     * Returns all loans, including the ones
+     * where the book has been already returned.
```

Comment:
In fact, it returns ALL books

Reply:
I don't get it. The return value is `List<SingleLoan>`.

Reply:
Sorry, I meant it ruturns all SingleLoans. And I read it as excluding. So te Java doc is fine. But please add returnedAt attribute to the SingleLoan.

Reply:
The SingleLoans now contains the `returnedAt` attribute.